src/cascadia/TerminalApp/App.xaml

```diff
@@ -54,17 +54,17 @@
                     <ResourceDictionary.ThemeDictionaries>
                         <ResourceDictionary x:Key="Dark">
                             <!--  Define resources for Dark mode here  -->
-                            <SolidColorBrush x:Key="TabViewBackground"
-                                             Color="#FF333333" />
+                            <StaticResource x:Key="TabViewBackground"
+                                            ResourceKey="ApplicationPageBackgroundThemeBrush" />
 
                             <SolidColorBrush x:Key="UnfocusedBorderBrush"

```

Comment:
can you change both of the UnfocusedBorderBrush resources as well? they should match the title bar. you can validate them by splitting a pane

Reply:
So it works for _closing_ a pane, but for opening a pane the background is actually _transparent_ (this issue exists in main)